src/popover/internal.tsx

```diff
@@ -27,6 +27,7 @@ export interface InternalPopoverProps extends Omit<PopoverProps, 'triggerType' |
   __onOpen?: NonCancelableEventHandler<null>;
   triggerType?: PopoverProps.TriggerType | 'filtering-token';
   size: PopoverProps.Size | 'content';
+  __closeAnalyticsAction?: string;
```

Comment:
Shouldn't popover emit its own metric on dismiss, why is it coming from the property filter?

Reply:
Internal components do not emit their own metrics because consumers do not explicitly add the internal component, so the metric would generate noise.